lib/src/util.dart

```diff
@@ -1,4 +1,7 @@
 import 'package:fixnum/fixnum.dart';
+import 'package:intl/intl.dart';
```

Comment:
where is intl used?

Reply:
yea the name is totally not related. DateFormat uses it to [parse string dates into DateTime](https://github.com/googleinterns/dv360-excel-plugin/pull/64/files/7e4b88f8be83899337522a9b8b62f178bf710882#diff-1f3f8d698cd9e4a97d005c251f4844d3R28-R30).